src/tests/utils/constants.cairo

```diff
@@ -18,7 +18,9 @@ const SALT: felt252 = 'SALT';
 const SUCCESS: felt252 = 123123;
 const FAILURE: felt252 = 456456;
 const MIN_TRANSACTION_VERSION: felt252 = 1;
-// 2**128 + TRANSACTION_VERSION
+// 2**128
+const QUERY_OFFSET: felt252 = 0x100000000000000000000000000000000;
+// QUERY_OFFSET + MIN_TRANSACTION_VERSION
 const QUERY_VERSION: felt252 = 0x100000000000000000000000000000001;
```

Comment:
should we make this query + offset?

Reply:
Changing the var name or what it's bound to? If the latter, we can't change `0x100000000000000000000000000000001` to offset + version because only literal constants are supported (though, that'd be preferable)

Reply:
gotch